orchestration/Chart.yaml

```diff
@@ -10,6 +10,6 @@ dependencies:
     repository: https://broadinstitute.github.io/monster-xml-to-json-list
     alias: xmlToJsonList
   - name: argo-templates
-    version: ~0.0.2
+    version: ~0.0.2-DSPDC-950
```

Comment:
Ohhhh I wonder if the changes didn't stick because of the `~` in the front. I'd remove it before you merge.